doc/rados/operations/user-management.rst

```diff
@@ -45,118 +48,120 @@ Ceph interprets the command like this:
 `Architecture - High Availability Authentication`_.
 
 Background
 ==========
 
-Irrespective of the type of Ceph client (e.g., Block Device, Object Storage,
-Filesystem, native API, etc.), Ceph stores all data as objects within `pools`_.
-Ceph users must have access to pools in order to read and write data.
-Additionally, Ceph users must have execute permissions to use Ceph's
-administrative commands. The following concepts will help you understand Ceph
-user management.
+No matter what type of Ceph client is used (for example: Block Device, Object
+Storage, Filesystem, native API), Ceph stores all data as objects within
+`pools`_.  Ceph users must have access to pools in order to read and write
+data, and Ceph users must have execute permissions in order to use Ceph's
+administrative commands. The following concepts will help you understand
+Ceph['s] user management.
 
 User
 ----
 
-A user is either an individual or a system actor such as an application.
+A user is either an individual or a system actor (for example, an application).
 Creating users allows you to control who (or what) can access your Ceph Storage
-Cluster, its pools, and the data within pools.
+Cluster, its pools, and the data within those pools.
 
-Ceph has the notion of a ``type`` of user. For the purposes of user management,
-the type will always be ``client``. Ceph identifies users in period (.)
-delimited form consisting of the user type and the user ID: for example,
+Ceph has the concept of a ``type`` of user. For purposes of user management,
+the type will always be ``client``. Ceph identifies users in a "period-
+delimited form" that consists of the user type and the user ID: for example,
 ``TYPE.ID``, ``client.admin``, or ``client.user1``. The reason for user typing
-is that Ceph Monitors, OSDs, and Metadata Servers also use the Cephx protocol,
-but they are not clients. Distinguishing the user type helps to distinguish
-between client users and other users--streamlining access control, user
-monitoring and traceability.
+is that the Cephx protocol is used not only by clients but also non-clients,
+such as Ceph Monitors, OSDs, and Metadata Servers. Distinguishing the user type
+helps to distinguish between client users and other users--and this distinction
+streamlines access control, user monitoring, and traceability.
 
-Sometimes Ceph's user type may seem confusing, because the Ceph command line
+Sometimes Ceph's user type might seem confusing, because the Ceph command line
 allows you to specify a user with or without the type, depending upon your
 command line usage. If you specify ``--user`` or ``--id``, you can omit the
-type. So ``client.user1`` can be entered simply as ``user1``. If you specify
-``--name`` or ``-n``, you must specify the type and name, such as
-``client.user1``. We recommend using the type and name as a best practice
-wherever possible.
+type. For example, ``client.user1`` can be entered simply as ``user1``. On the
```

Comment:
Suggest above s/can/may/ twice.

Above Below "specify" is used twice, suggest:

>When specifying \`\`--user\`\` or \`\`--id\`\`, you may omit the type, entering \`\`client.user1\`\` as simply \`\`user1\`\`.  However, when specifying  \`\`--name`\` or ``-n\``, you must supply both the type and the name, for example: `\`client.user1\``.  We recommend supplying both the type and the name as a best practice whenever possible.

Reply:
"supplying" is good.